src/opset/Conv_11.c

```diff
@@ -22,133 +22,44 @@
 #include <connx/connx.h>
 
 /*{% for DTYPE, TYPE in loop_types(FLOAT32, FLOAT64) %}*/
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
This is duplicated for all of `DTYPE` loop.
Move it above the template for-loop line

Reply:
Done